features/bevragen/persoon/adressering/adres-regels/dev/fields-gba.feature

```diff
@@ -0,0 +1,432 @@
+    | 080910                  | gemeente van inschrijving          | adressering.adresregel2                                                                  |
+    | 081000                  | hele groep adreshouding            | adressering.adresregel3                                                                  |
+    | 081010                  | functie adres                      | adressering.land                                                                         |
+    | 081030                  | datum aanvang adreshouding         | adressering.adresregel1,adressering.adresregel2                                          |
+    | 081100                  | hele groep adres                   | adressering.adresregel1,adressering.adresregel3                                          |
+    | 081110                  | straatnaam                         | adressering.adresregel1,adressering.land                                                 |
+    | 081115                  | naam openbare ruimte               | adressering.adresregel2,adressering.adresregel3                                          |
+    | 081120                  | huisnummer                         | adressering.adresregel2,adressering.land                                                 |
+    | 081130                  | huisletter                         | adressering.adresregel3,adressering.land                                                 |
+    | 081140                  | huinummertoevoeging                | adressering.adresregel1                                                                  |
+    | 081150                  | aanduiding bij huisnummer          | adressering.adresregel2                                                                  |
+    | 081160                  | postcode                           | adressering.adresregel3                                                                  |
+    | 081170                  | woonplaatsnaam                     | adressering.land                                                                         |
+    | 081180                  | identificatiecode verblijfplaats   | adressering.adresregel1,adressering.adresregel2                                          |
+    | 081190                  | identificatiecode nummeraanduiding | adressering.adresregel1,adressering.adresregel3                                          |
+    | 088500                  | hele groep geldigheid              | adressering.adresregel1,adressering.land                                                 |
+    | 088510                  | datum ingang geldigheid            | adressering.adresregel2,adressering.adresregel3                                          |
+
+Rule: wanneer één of meerdere adresregel velden voor een locatie wordt gevraagd, dan wordt altijd de waarde van 'locatiebeschrijving (12.10)' geleverd
```

Comment:
Bedoel je: wanneer één of meerdere adresregel velden worden gevraagd voor een persoon met een locatie als verblijfplaats wordt gevraagd, dan wordt altijd de waarde van 'locatiebeschrijving (12.10)' geleverd?